lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -713,15 +713,17 @@ FIRRTLModuleLowering::lowerModule(FModuleOp oldModule, Block *topLevelModule,
   if (failed(lowerPorts(firrtlPorts, ports, oldModule, loweringState)))
     return {};
 
-  loweringState.processRemainingAnnotations(oldModule,
-                                            AnnotationSet(oldModule));
   // Build the new hw.module op.
   auto builder = OpBuilder::atBlockEnd(topLevelModule);
   auto nameAttr = builder.getStringAttr(oldModule.getName());
   auto newModule =
       builder.create<hw::HWModuleOp>(oldModule.getLoc(), nameAttr, ports);
   if (auto outputFile = oldModule->getAttr("output_file"))
     newModule->setAttr("output_file", outputFile);
+  if (AnnotationSet::removeAnnotations(oldModule, dutAnnoClass))
+    newModule->setAttr("DesignUnderTest", builder.getBoolArrayAttr(true));
```

Comment:
Since I don't think that this will ever be false, just not set, would it make sense to use a unit attr here?

```suggestion
    newModule->setAttr("DesignUnderTest", builder.getUnitAttr();
```